playbooks/viya-mmsu/viya-services-enable.yml

```
@@ -0,0 +1,25 @@
+####################################################################
+#### viya-services-enable.yml                                  ####
+####################################################################
+#### Author: SAS Institute Inc.                                 ####
+####################################################################
+#
+# Copyright (c) 2025, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+- import_playbook: ../common/handle_hostgroup_hyphens.yml
+  tags:
+    - always
+
+- hosts: sas_all
+  become: yes
+  become_user: root
+  gather_facts: False
+  any_errors_fatal: true
+
+  tasks:
+    - name: Enable SAS Viya all services from system reboot
```

minor grammar correction suggestion:
```suggestion
    - name: Enable SAS Viya all services at system reboot
```

---

Same grammar used in disable playbook, so will include same change there for consistency.